package.json

```diff
@@ -6,33 +6,35 @@
     "build-dev": "vite build --mode=development"
   },
   "devDependencies": {
-    "@popperjs/core": "^2.11.6",
-    "@tailwindcss/forms": "^0.5.2",
-    "@vitejs/plugin-vue": "^4.0.0",
-    "alpinejs": "^3.4.2",
-    "autoprefixer": "^10.4.2",
-    "axios": "^1.1.2",
-    "bootstrap": "^5.2.3",
+    "@popperjs/core": "^2.11.8",
+    "@tailwindcss/forms": "^0.5.7",
+    "@vitejs/plugin-vue": "^5.0.4",
+    "alpinejs": "^3.13.7",
+    "autoprefixer": "^10.4.19",
+    "axios": "^1.6.8",
+    "bootstrap": "^5.3.3",
     "laravel-vite-plugin": "^1.0.2",
-    "lodash": "^4.17.19",
-    "postcss": "^8.4.6",
-    "sass": "^1.56.1",
-    "vite": "^5.2.4",
+    "lodash": "^4.17.21",
+    "postcss": "^8.4.38",
+    "sass": "^1.72.0",
+    "vite": "^5.2.6",
     "vite-plugin-pwa": "^0.19.7"
   },
   "dependencies": {
-    "@fontsource/nunito": "^4.5.12",
-    "@fortawesome/fontawesome-free": "^6.2.1",
-    "chart.js": "^4.2.0",
+    "@fontsource/nunito": "^5.0.17",
+    "@fortawesome/fontawesome-free": "^6.5.1",
+    "@jridgewell/sourcemap-codec": "^1.4.15",
+    "@rollup/plugin-terser": "^0.4.4",
+    "chart.js": "^4.4.2",
     "flatpickr": "^4.6.13",
-    "jquery": "^3.6.3",
-    "laravel-vue-i18n": "^2.3.2",
-    "sweetalert2": "^11.7.0",
+    "jquery": "^3.7.1",
+    "laravel-vue-i18n": "^2.7.6",
+    "sweetalert2": "^11.10.7",
     "vform": "^2.1.2",
     "vuejs-paginate-next": "^1.0.2",
-    "workbox-precaching": "^6.5.4",
-    "workbox-routing": "^6.5.4",
-    "workbox-window": "^6.5.4"
```

Comment:
Jeg er virkeligt i tvivl om den her ændring 🤔 

Kan du merge develop ind i branchen igen ? Jeg tænker mest på  om der kan være conflicts med Renovate?

Umiddelbart ser det fint ud at opdatere versionerne mest muligt.

If it works - it works! 🎉

Reply:
@negoziator 
Det kunne være, jeg skulle have beskrevet bedre, hvad jeg var ude på.
Ja, jeg opdaterede alle npm-pakker mest muligt fordi.... det havde jeg lyst til :)
Og så har jeg droppet, at de skal være i devDependencies. Det gør jeg fordi når prod-serveren deployer, så kører den npm run build, og så forstår den jo ikke i prod, at der mangler en hel masse, som den bruger til at bygge med.
Giver det mening?
Ellers lad os lige summe over det sammen over slack en af dagene :)

Reply:
@negoziator Ping :)